XamlControlsGallery/ControlPages/NumberBox.xaml

```diff
@@ -0,0 +1,63 @@
+﻿<!--
+//*********************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+// THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF
+// ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY
+// IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR
+// PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.
+//
+//*********************************************************
+-->
+<Page x:Class="AppUIBasics.ControlPages.NumberBoxPage" 
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" 
+        xmlns:local="using:AppUIBasics"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+        mc:Ignorable="d">
+
+    <StackPanel HorizontalAlignment="Left">
+
+        <local:ControlExample HeaderText="A NumberBox that evaluates expressions." 
+                              XamlSource="NumberBox/NumberBoxSample1_xaml.txt">
+            <local:ControlExample.Example>
+                <muxc:NumberBox Header="Enter an expression:" Value="NaN" PlaceholderText="1 + 2^2" AcceptsExpression="True" />
+            </local:ControlExample.Example>
+        </local:ControlExample>
+
+        <local:ControlExample HeaderText="A NumberBox with a spin button." 
+                              XamlSource="NumberBox/NumberBoxSample2_xaml.txt">
+            <local:ControlExample.Example>
+                <StackPanel>
+                    <muxc:NumberBox x:Name="NumberBoxSpinButtonPlacementExample"
+                                    Header="Enter an integer:" 
+                                    Value="10" 
+                                    SpinButtonPlacementMode="Compact" 
+                                    SmallChange="10"
+                                    LargeChange="100">
```

Comment:
Same request as in the code sample: please close the NumberBox inline instead of having the hanging `</muxc:NumberBox>` on its own line.